VERSION

```diff
@@ -1,4 +1,4 @@
-VERSION="2.8.4"
+VERSION="2.9.0"
```

Comment:
should not target 2.9 branch, do not forget to create it and retarget PR before merging.

(though we should not release zenko 2.9.0 right after : if you need to create a release afterwards for bumping in Artesca, should be a pre-release like `2.9.0-preview.1` or `2.9.0-alpha.1`)

Reply:
Yes, this is only for testing.

Reply:
PR is now rebased on the latest dev/2.9 branch.